autode/transition_states/locate_tss.py

```diff
@@ -299,6 +300,10 @@ def get_ts(reaction, reactant, bond_rearr, is_truncated=False):
         logger.warning('Reaction had no complexes - generating')
         reaction.find_complexes()
 
+    if bond_rearr.n_fbonds > bond_rearr.n_bbonds:
+        raise NotImplementedError('Cannot treat more forming than breaking '
+                                  'bonds, reverse the reaction(?)')

```

Comment:
The reactants/products will get swapped in reaction.py, won't they? So swapping them around won't change the order of the reactants & products in get_ts(). I may be misunderstanding this though...

Reply:
they will, but this is for if the get_ts function is called separately e.g. https://duartegroup.github.io/autodE/examples/tss.html